packages/aws-rfdk/lib/core/lib/exporting-log-group.ts

```diff
@@ -60,11 +60,20 @@ export interface ExportingLogGroupProps {
  *
  * Resources Deployed
  * ------------------------
- * 1) The Lambda SingletonFunction that checks for the existence of the LogGroup;
- * 2) The CloudWatch LogGroup (if it didn't exist already);
- * 3) The CloudWatch Alarm watching log exportation failures;
- * 4) The CloudWatch Event Rule to schedule log exportation;
- * 5) The Lambda SingletonFunction, with role, to export log groups to S3 by schedule.
+ * - The Lambda SingletonFunction that checks for the existence of the LogGroup;
+ * - The CloudWatch LogGroup (if it didn't exist already);
+ * - The CloudWatch Alarm watching log exportation failures;
+ * - The CloudWatch Event Rule to schedule log exportation;
+ * - The Lambda SingletonFunction, with role, to export log groups to S3 by schedule.
```

Comment:
minor style inconsistency nit: seems that we sometimes add semi-colons after each resource in the list and sometimes we use periods.

Reply:
Technically, the semi-colons are proper, but periods seem way more common. I'll make it consistent.